.github/workflows/build.yaml

```diff
@@ -40,34 +41,33 @@ jobs:
 
       # Generate .vsix package for the extension 
       - name: Generate .vsix package
-        if: runner.os == 'Linux'  
         run: |
           npm install -g @vscode/vsce
           vsce package
 
-      # Upload VSIX artifact 
+      # Upload VSIX artifact
       - name: Upload VSIX artifact
-        if: runner.os == 'Linux'  
```

Comment:
no, see previous comment

Reply:
Linux support has been re-added for generating and uploading artifact